torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -131,13 +131,17 @@ std::string nodeName(Node * n) {
 
 // TODO: we need to support double-precision
 std::unordered_map<NodeKind,std::function<std::string(Node*)>> simple_map_ops = {
-  {kSigmoid,         [](Node*) { return "1.f / (1.f + expf(-${0}))"; }},
-  {kTanh,            [](Node*) { return "tanhf(${0})"; }},
-  {kMul,             [](Node*) { return "${0} * ${1}"; }},
-  {kAdd,             [](Node*) { return "${0} + ${1}"; }},
-  {kNeg,             [](Node*) { return "(-${0})"; }},
-  // TODO: support both float and int constants
-  {kAddConstant,     [](Node* n) { return std::to_string(n->f(kvalue)) + " + ${0}"; }},
+  {ksigmoid,         [](Node*) { return "1.f / (1.f + expf(-${0}))"; }},
+  {ktanh,            [](Node*) { return "tanhf(${0})"; }},
+  {kmul,             [](Node*) { return "${0} * ${1}"; }},
+  {kadd,             [](Node*n) -> std::string {
+    if(n->inputs().size() == 2)
+      return "${0} + ${1}";
+    else // TODO: support both float and int constants
+      return "${0} + " + std::to_string(n->f(kvalue));
```

Comment:
Are you sure this works? IIRC, `value` is recorded as a scalar `z` not a float `f`.